tests/unit/workflow/test_workflow.py

```diff
@@ -107,3 +107,11 @@ def plus_one(x: int = 0) -> int:
             return x + 1
 
         self.assertEqual(plus_one().outputs.y.value, 1)
+
+    def test_working_directory(self):
+        wf = Workflow("wf")
+        self.assertTrue(wf._working_directory is None)
+
```

Comment:
```suggestion
self.assertIsInstance(wf.working_directory, DirectoryObject)
self.assertTrue(wf.working_directory.path.absolute().endswith(wf.label)

```

Maybe the endswith needs a "/" or something? Also I guess we need similar tests for `Node`, where it starts as none, raises an error when asked for, and then passes the same endswith test after a parent is added